tests/SoldierAPI.test.ts

```diff
@@ -0,0 +1,46 @@
+import request from 'supertest';
+import router from '../src/routes/routes';
+import Soldier from '../src/repository/soldierRepo';
+import * as soldiers from './testData';
+
+const app = router;
+beforeEach(async () => {
+  await Soldier.deleteMany();
+});
+describe('Insert Soldier into DB', () => {
+  it('Should Send an API Post Request and save soldier to DB, Expected 201:', async () => {
```

Comment:
you are describing what the test does, describe the behavior of the code that you are testing for:

```
it('should insert soldier and return 201', async () => {
```